app/code/Magento/CustomerGraphQl/Controller/HttpRequestProcessor/CustomerProcessor.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CustomerGraphQl\Controller\HttpRequestProcessor;
+
+use Magento\Authorization\Model\UserContextInterface;
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Customer\Api\Data\GroupInterface;
+use Magento\Customer\Model\Context;
+use Magento\Customer\Model\GroupManagement;
+use Magento\GraphQl\Controller\HttpHeaderProcessorInterface;
+use Magento\GraphQl\Model\Query\ContextFactoryInterface;
+use Magento\GraphQl\Model\Query\ContextInterface as GraphQlContext;
+use Magento\Framework\App\Http\Context as HttpContext;
+
+/**
+ * Process the "Authorization" header
+ */
+class CustomerProcessor implements HttpHeaderProcessorInterface
+{
+    /**
+     * @var GraphQlContext
+     */
+    protected $graphQlContext;
+
+    /**
+     * @var HttpContext
+     */
+    protected $httpContext;
+
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    protected $customerRepository;
+
+    /**
+     * Customer constructor.
```

Comment:
I would suggest removing this comment from DocBlock since it's redundant and only adds extra line to the code. Thank you.
```suggestion
```